Sources/Aardvark/Bug Reporting/ARKBugReportAttachment.h

```diff
@@ -21,7 +21,7 @@
 #import <Foundation/Foundation.h>
 
 
-@interface ARKEmailAttachment : NSObject
+@interface ARKBugReportAttachment : NSObject
```

Comment:
what's motivating the name change here? There are plenty of ways to report a bug don't involve email – I'm not convinced that this change is helpful just yet.

Reply:
I want to separate the concept of a bug report attachment from the email bug reporter, so that attachments can be added to _any_ bug report, whether it's filed by email or another method.

Reply:
Got it. I'm on-board. Making the attachment itself generic makes sense to me 🙂